### DISCUSSION TITLE
Ability to backup multiple folders for single game.

### DISCUSSION THREAD

**Discussion by RedRain77** [MikeMaximus/gbm#236] [Q&A] [answered]
Is there a way to backup multiple folders for a single game?
I've run into a few games that keep certain files in different folders (userconfig and save files).

This would normally be fine if everything was in the same parent folder, but there sometimes located in both the appdata folder and the default game folder.

**Comment by MikeMaximus** [comment, marked as answer]
You can use linked configurations to do this, any linked configuration will also trigger when doing a single backup or restore.

For example, you create two distinct configurations for the game, one for the save files and another for the userconfig files.  You can link configurations together by using the "Link Configuration..." option in the Game Manager, the option is available in the Link Options (Chain Icon) sub menu.

The "Link Configuration" sub-section in the [Game Manager portion of the manual](https://mikemaximus.github.io/gbm-web/manual.html#gamemanager) has more details on how this works.

One note I keep forgetting to add to the manual is that only the parent configuration needs to actually detect the game to trigger a backup, you can leave the "Process" field blank in any of the linked configurations.

I'm going to move this issue to the discussion section, feel free to reply if you have any other questions.
